app/Http/Controllers/DashboardController.php

```diff
@@ -16,13 +19,62 @@ public function __construct()
         $this->middleware('auth');
     }
 
+    public function store(Request $req)
+    {
+        $newExpense = new Expense;
+        /*TODO Set name of owner to not hardcoded*/
+        $newExpense->owner_username= Auth::user()->username;
+        $newExpense->amount=$req->expAmount;
+        $newExpense->type=$req->expType;
+        $newExpense->date=$req->expDate;
+        $newExpense->comments=$req->expComments;
```

Comment:
Just double checking if we need some sort of validation here. Because we're assigning the request fields right away